qt/applications/workbench/workbench/config/fonts.py

```
@@ -67,3 +71,36 @@ def text_font():
         _TEXT_FONT_CACHE.setFixedPitch(True)
 
     return _TEXT_FONT_CACHE
+
+
+# Returns if the user is using a dark theme on macOS
+def _is_theme_dark():
+    result = subprocess.run(
+        ["defaults", "read", "-g", "AppleInterfaceStyle"],
```

Ruff (0.15.10)

[error] Starting a process with a partial executable path

(S607)